<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="startWavlengthLabel">
        <property name="text">
         <string>Start (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="startWavlengthDoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1750.000000000000000</double>
        </property>
        <property name="value">
         <double>1500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stopWavelengthLabel">
        <property name="text">
         <string>Stop (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stopWavelengthDoubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1750.000000000000000</double>
        </property>
        <property name="value">
         <double>1600.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sensitivityLabel">
        <property name="text">
         <string>Sensitivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sensitivityComboBox">
        <item>
         <property name="text">
          <string>Hold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="referenceLevelLabel">
        <property name="text">
         <string>Reference Level (dBm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="referenceLevelDoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resoltuionNmLabel">
        <property name="text">
         <string>Resoltuion (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="resoltuionNmDoubleSpinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PointsNmlabel">
        <property name="text">
         <string>Points/nm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PointsNmspinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SweepPushButton">
        <property name="text">
         <string>Sweep</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/icons8-play-50.png</normaloff>Icons/icons8-play-50.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
      <item>
       <widget class="PlotWidget" name="plotWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Visible</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="NewGroupPushButton">
              <property name="text">
               <string>New Group</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>Icons/addition-color-icon.png</normaloff>Icons/addition-color-icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="DeletePushButton">
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>Icons/delete-icon.png</normaloff>Icons/delete-icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SavePushButton">
              <property name="text">
               <string>Save checked</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>Icons/icons8-save-96.png</normaloff>Icons/icons8-save-96.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
